src/openqa_client/client.py

```diff
@@ -377,3 +377,34 @@ def get_jobs(
             # there as it only considers the jobs queried.
             jobdicts = self.find_clones(jobdicts)
         return jobdicts
+
+    def get_latest_build(
+        self,
+        group_id: int,
+        all_passed: bool = True,
+        sort_key: Callable = int,
+        default_return: str = "",
```

Comment:
can we just get rid of this now? AFAICS it isn't really necessary given you can use the `build = client.get_latest_build(42) or "foo"` approach I suggested before. Removing the arg makes the code shorter and means we don't have to test it.

Reply:
done